tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -449,6 +449,14 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string RemoveClientServiceNamesEnabled = "DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED";
 
+        /// <summary>
+        /// Configuration key for the comma-separated list of user disabled
+        /// ADO.NET CommandType names that should not have Span created for them.
+        /// <para>"InterceptableDbCommand" and "ProfiledDbCommand" are always disabled by default.</para>
+        /// </summary>
+        /// <seealso cref="TracerSettings.DisabledAdoNetCommandTypes"/>
+        public const string DisabledAdoNetCommandTypes = "DD_TRACE_DISABLED_ADONET_COMMAND_TYPES";
```

Comment:
Level of care low, but what do you think of using `IGNORED` instead of `DISABLED`? As in `DD_TRACE_IGNORED_ADONET_COMMAND_TYPES`. Just throwing it out there, feel free to ignore.

Reply:
So I was between `IGNORED` or `DISABLED` and opted for `DISABLED` actually because the `Activity` stuff (totally irrelevant here though).

As I also have this PR: https://github.com/DataDog/dd-trace-dotnet/pull/5795 which adds `DD_TRACE_DISABLED_ACTIVITY_SOURCES` and for `Activity` we already have a concept of `Ignored` with the `IgnoreActivityHandler`: https://github.com/DataDog/dd-trace-dotnet/blob/38479234bcbc1d5270e638750f45bfa22f2177d0/tracer/src/Datadog.Trace/Activity/Handlers/IgnoreActivityHandler.cs#L16


Maybe I'm overthinking it?

Reply:
Oh yeah, it's nice to be consistent then with `DISABLED` in both. Thanks!